test/e2e/util.go

```diff
@@ -234,6 +234,26 @@ func SkipUnlessNodeCountIsAtLeast(minNodeCount int) {
 	}
 }
 
+func SkipIfMajorMinorVersionIs(c client.VersionInterface, major, minor string) {
```

Comment:
This should be `SkipIfVersionLessThan` or something similar, because we won't want to enumerate both 1.0 and 1.1 for tests we start implementing for 1.2.  (Same below)